cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -244,12 +253,17 @@ impl YaccParser {
                 continue;
             }
             if let Some(j) = self.lookahead_is("%expect", i) {
-                if self.ast.expect.is_some() {
-                    return Err(self.mk_error(YaccParserErrorKind::DuplicateExpectDeclaration, i));
-                }
                 i = self.parse_ws(j, false)?;
                 let (j, n) = self.parse_int(i)?;
-                self.ast.expect = Some(n);
+                let span = Span::new(i, j);
```

Comment:
So this is referring to the span surrounding `"1"` in `"%expect 1"`.
Up until this point this has been the right thing, but because it is the expect declaration which is unique rather than the value.
This should span "%expect 1" or "%expect".

Reply:
Cool, then I won't change anything.